### ISSUE TITLE
jQuery image field

### ISSUE THREAD

**Issue by benoitboucart** [genemu/GenemuFormBundle#212]
Hi

I was trying the "image field type" but I get this error:

Notice: Undefined index: folder in //vendor/genemu/form-bundle/Genemu/Bundle/FormBundle/Form/JQuery/Type/FileType.php line 57

Form type: ->add('file', 'genemu_jqueryimage')

Version 2.1

Thanks

**Comment by tmueller91** [comment]
This worked for me:
```
$builder->add('picture', 'genemu_jqueryimage', [
            'configs' => [
                'folder' => 'uploads'
            ]
        ]);
```

**Comment by kmehmety** [comment]
Yes it should be work. Because you defined "config" in php. Also you can
define yml.

2015-03-01 2:55 GMT+02:00 tmueller91 <notifications@github.com>:
